tests/integration/questionnaire/test_questionnaire_radio_voluntary.py

```diff
@@ -12,3 +12,24 @@ def test_radio_voluntary(self):
         self.post(action="clear_radios")
         self.post()
         self.assertNotInBody("Coffee")
+
+
+class TestQuestionnaireRepeatingSectionRadioVoluntary(IntegrationTestCase):
```

Comment:
I realise that it matches the pattern above, but out of context and without reading the test I can't tell what it is testing. It is just clearing radio options? With the class name this could be as simple as test_clear_radios. I bit of space between the actions and the asserts also makes it a bit easy to digest

Reply:
Renamed and added some spacing, thanks!